### PR TITLE
Fix/auth for wp export

### PR DESCRIPTION
**Pull request by manwithtwowatches**
https://www.openproject.org/work_packages/10435

### CODE REVIEW

**Comment by myabc** [comment]
Travis is timing out on the cukes, but I think this is ok – @tilt can you take a look?

**Comment by tilt** [comment]
quite a lot of stuff, but quite intuitive.

File: public/templates/work_packages.list.html [hunk 1]
```diff
@@ -64,22 +64,22 @@
     <li><a href ng-click="toggleDisplaySums()"><i ng-if="query.displaySums" class="icon-action-menu icon-yes"></i><i ng-if="!query.displaySums" class="icon-action-menu no-icon"></i>{{ I18n.t('js.toolbar.settings.display_sums') }}</a></li>
     <li class="dropdown-divider"></li>
     <li><a href ng-click="saveQuery($event)"
-           ng-class="{'inactive': (!query.isNew() && cannot(query, 'update')) || (query.isNew() && cannot(query, 'create'))}">
+           ng-class="{'inactive': (!query.isNew() && cannot('query', 'update')) || (query.isNew() && cannot('query', 'create'))}">
```
Comment (tilt) [inline public/templates/work_packages.list.html:67]: did this work before when the object was passed to the service?

Comment (tilt) [inline public/templates/work_packages.list.html:67]: ok it didn't

**Pull request merged**